frontend/src/components/pages/CreateReview/CreateReviewPage.tsx

```diff
@@ -78,6 +115,62 @@ const CreateReview = (): React.ReactElement => {
     addBook(book);
   };
 
+  /**
+   * Function to be called when the user clicks the publish button.
+   */
+  const handlePublish = () => {
+    setReviewError(false);
+    setBylineError(false);
+
+    // Check if all fields are filled in
+    if (review !== "" && review !== "<p><br></p>" && byline !== "") {
+      setShowPublishModal(true);
+    } else {
+      if (review === "" || review === "<p><br></p>") {
+        setReviewError(true);
+      }
+      if (byline === "") {
+        setBylineError(true);
+      }
+    }
+  };
+
+  /**
+   * Function to be called when the review is published.
+   */
+  const onPublish = () => {
+    // check if all fields have been filled in
+    if (review !== "" || byline !== "" || books.length !== 0) {
+      // publish review
+      if (authenticatedUser?.id) {
+        reviewAPIClient
+          .publishReview({
+            body: review,
+            byline,
+            featured: featured === "1",
+            createdBy: parseInt(authenticatedUser?.id, 10),
+            publishedAt: new Date().getTime(),
```

Comment:
according to official docs: `Date.prototype.getTime()` Returns the numeric value of the specified date as the number of milliseconds since January 1, 1970, 00:00:00 UTC. (Negative values are returned for prior times.)